.circleci/config.yml

```diff
@@ -1471,7 +1496,7 @@ workflows:
           type: approval
           filters:
             branches:
-              ignore: /.*/
+              ignore: /v.*/
```

Comment:
Is this for testing purposes? Otherwise it does not make sense

Reply:
yes will be reverted after simulations